client/my-sites/domains/domain-management/edit/mapped-domain.jsx

```diff
@@ -152,6 +157,22 @@ const MappedDomain = createReactClass( {
 			</VerticalNavItem>
 		);
 	},
+
+	transferMappedDomainNavItem() {
+		const domain = this.props.domain;
```

Comment:
This is definitely a nitpick, but in case it's helpful/interesting: the usual pattern is using a [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment). It's slightly shorter, but also allows pulling in more stuff from `props` (which is usually the case). And it's more "future-proof" in case future PRs need additional stuff from props.

So, for example:

```
const { domain, translate } = this.props; // now you can use `translate` below, instead of `this.props.translate`
```

Up to you what props need to be shortened like this - the usual rule of thumb is if it's used more than once, do it. And if you're already destructuring props, pull in `translate` as well, even if it's used once.